### ISSUE TITLE
fade out would be nice

### ISSUE THREAD

**Issue by mrosseel** [marcorinck/angular-growl#3]
if there is a timeout chosen, out would be a nice visual effect to fade out the message. This could be an option such as globalTimeToLive: -1 for no fade, a time parameter for how fast the fadeout should happen.

**Comment by marcorinck** [comment]
Nice idea. Will do. 

Though will do it only for angularJs 1.2 as I would want to use $animate service. So no release for that before 1.2 is released.

**Comment by mrosseel** [comment]
sounds like a good idea to do it with $animate, looking forward to it.

**Comment by marcorinck** [comment]
I just committed the ngAnimate branch and have to say ... wow, its damn easy to do animations with angularJS 1.2!

Can you have a look and tell me what you think?

As this change breaks nothing when using only angular 1.0.x, I'll release it soon if you think its ok.

**Comment by mrosseel** [comment]
Hi,

I tested, a few remarks:
- the build dir files are not yet updated (I built them using 'grunt build:core')
- I see the animation when the message appears, but not when leaving? I'm using chrome and the latest angular/ngAnimate (1.2.0-rc2)

**Comment by marcorinck** [comment]
thank you for testing!

yes I know build files are only built when doing release, but the demo is updated and should be working.

I just fixed the missing animation.

The leaving animations was in fact missing, when there was only one message left to diplay, but was working when there were at least two messages.

The problem came from the "ngShow" directive of the parent element, which sets display to "none" which can not be animated. I removed ngShow from the parent element completely. It was not necessary as it does not contain anything when there are no messages. 

So, now its working even when there is only one message.
